.lando.base.yml

```diff
@@ -14,9 +14,18 @@ env_file:
   - .env
 services:
   appserver:
+    scanner: false
```

Comment:
This stops Lando from sitting waiting for an HTTP response from every website you run. Usually you don't care (if it's broken it'll be obvious) so why spend ages waiting